strapi/strapi
# chore(database): refactor database schema to use naming functions

### What does it do?

Use the naming functions in the actual schema creation

### Why is it needed?

Final piece to refactoring all known* hardcoded strings to use naming functions

* there are almost certainly other places in Strapi using hardcoded strings for identifiers; but it's hard to find them until we are far enough along to turn on the shortening feature and see. Anything missing will be addressed at that time, before merging any actual breaking changes.

### How to test it?

This is just another refactoring with no effect. So like with the other related PRs; create a complex data set (including relations, dz, components, etc) without this PR, then switch to this PR and absolutely nothing tangible should change. All your data should still be there, all database identifiers should still be the same.

Tests covering 100% of lines/branches with identifiers in them have been added to prove this is the case

### Related issue(s)/PR(s)

https://github.com/strapi/strapi/pull/19567
https://github.com/strapi/strapi/pull/19561
https://github.com/strapi/strapi/pull/19460
